MatricesForHomalg/gap/HomalgRing.gi

```diff
@@ -2891,6 +2891,17 @@ InstallMethod( ViewObj,
     
 end );
 
+##
+InstallMethod( ViewString,
+        "for homalg ring elements",
+        [ IsHomalgRingElement ],
+        
+  function( o )
```

Comment:
I would prefer such a code

```
InstallMethod( ViewString,
        "for homalg ring elements",
        [ IsHomalgRingElement ],

  Name );
```

Reply:
Makes sense, I have pushed an updated version. Although I really hate it if I quickly want to put a "Display" or an "Error" somewhere for debugging and come across a method installation like this :D